src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.connection.e2e.spec.ts

```diff
@@ -121,37 +120,6 @@ void describe('MongoDBEventStore connection', () => {
     }
   });
 
-  void it('does not disconnect connected client', async () => {
```

Comment:
**Note:** It's nice that we can remove tests; it's no longer needed, as this use case won't even be allowed.